Dockerfile

```
@@ -0,0 +1,37 @@
+#
+# BUILDER
+#
+FROM docker.io/library/golang:1.22.3 AS builder
+
+WORKDIR /buildroot
+
+COPY go.mod go.mod
+COPY go.sum go.sum
+RUN go mod download
+
+# Copy the source.
+COPY pkg/ pkg/
+COPY cmd/ cmd/
+
+# Disable CGO.
+ENV CGO_ENABLED=0
+
+# Build the image.
+RUN go build -o build/sample-cosi-driver cmd/sample-cosi-driver/*.go
+
+#
+# FINAL IMAGE
+#
+FROM gcr.io/distroless/static:latest AS runtime
+
+LABEL maintainers="Kubernetes Authors"
+LABEL description="COSI Sample Driver"
```

nit: Those two labels should be prefixed with `org.opencontainers.image`:

```suggestion
LABEL org.opencontainers.image.maintainers="Kubernetes Authors"
LABEL org.opencontainers.image.description="COSI Sample Driver"
```